src/bin/pg_autoctl/pgctl.c

```diff
@@ -448,7 +456,16 @@ config_find_pg_ctl(PostgresSetup *pgSetup)
 		 */
 		PostgresSetup pgSetupFromPgConfig = { 0 };
 
-		log_debug("Failed to find pg_ctl in PATH, looking for pg_config");
+		if (pg_ctls.found == 0)
+		{
+			log_debug("Failed to find pg_ctl in PATH, looking for pg_config");
+		}
+		else
+		{
+			log_debug("Found %d entries for pg_ctl in PATH, "
+					  "looking for pg_config",
+					  pg_ctls.found);
+		}
 
 		if (set_pg_ctl_from_pg_config(&pgSetupFromPgConfig))
```

Comment:
I would expect that `set_pg_ctl_from_pg_config` would also use `search_path_deduplicate_symlinks`, but I don't see it in the diff.

Reply:
Added now, good point.